src/hotspot/share/gc/shenandoah/shenandoahNumberSeq.cpp

```diff
@@ -56,7 +56,7 @@ void HdrSeq::add(double val) {
   int mag;
   if (v > 0) {
     mag = 0;
-    while (v > 1) {
+    while (v >= 1) {
```

Comment:
You can safely ignore the changes in this file and the next.
They are part of a separate PR to tip, and will eventually get reconciled when tip is merged into the project repo.